projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -125,6 +144,7 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
     return this.formGroupErrors;
   }
 
+  // @todo could this be removed to avoid an override and just use `takeUntilDestroyed`?
```

Comment:
prob not due to the `emitNullOnDestroy`, we'd also have to require the developer to implement an empty `OnDestroy` which might be even more confusing

Reply:
At least it'd be safe :man_shrugging: It feels so dangerous to have an `ngOnDestroy` hook :smirk: ... I'd definitely forgot about it ha

Reply:
So enforcement would happen with a `public abstract ngOnDestroy(): void;` ?

Reply:
But wait... Why would we need devs to do that at all?
If they don't have the method, [it would be fine](https://github.com/cloudnc/ngx-sub-form/pull/41/files#diff-5796510f30fdd5bede9d709ce53ef225R51), wouldn't it?

If it's just a lint issue then I think we should probably turn it off on that line

Reply:
From memory angular requires the method to actually be present to be able to detect it and call the hook. There was some reason that we had to have `component: OnDestroy` as the param type to require it to implement OnDestroy. Might be worth double checking that is an actual requirement